app/assets/javascripts/controllers/container_dashboard/util/chart-data-mixin.js

```diff
@@ -0,0 +1,7 @@
+angular.module('miq.util').factory('ChartsDataMixin', [function chartDataMixinFactory () {
+  'use strict';
+  return {
+    dashboardHeatmapChartHeight:    286,
```

Comment:
The angular patternfly directive for heatmap takes a height for the size of the heatmap container and this height must be passed in to the directive so it really can't be in CSS. 

We could add an issue for angular patternfly not to set the height if not passed then the caller could set it via a css selector, but as of now setting the height in css will get overridden by the directive setting the height directly via a style setting.